code/datums/wires/vending.dm

```diff
@@ -9,7 +9,7 @@ var/const/VENDING_WIRE_IDSCAN = 8
 
 /datum/wires/vending/CanUse(var/mob/living/L)
 	var/obj/machinery/vending/V = holder
-	if(!istype(L, /mob/living/silicon))
+	if(!istype(L, /mob/living/silicon) && !L.lying && !L.incapacitated())
```

Comment:
Oh shit, you can avoid shocks if you're resting and the wirecutters are glued to your hands though.